src/contracts/addreses.ts

```diff
@@ -179,33 +179,6 @@ const tokens: Record<GebDeployment, TokenList> = {
             collateralAuctionHouse: '0x55f7885cc3ccaf68549d1bed30ae23482f2f76d9',
             isCollateral: true,
         },
-        TTM: {
-            address: '0xE3Efbd4fafD521dAEa38aDC6D1A1bD66583D5da4',
-            decimals: 0,
-            symbol: 'TTM',
-            bytes32String: TOTEM,
-            collateralJoin: '0x0946bcced96fa4d4d1a93da2f0de8308f83aa9e7',
-            collateralAuctionHouse: '0xd926a968e417b0d51ff33ba924b866782da6d1f8',
-            isCollateral: true,
-        },
-        HAI: {
-            address: '0x521f5663706b0B8322926Cdead069b57b484B81f',
-            decimals: 18,
-            symbol: 'HAI',
-            bytes32String: '',
-            collateralJoin: '0x34efB8C04665d67bd2325DFbBD5cc0b2009F8a55',
-            collateralAuctionHouse: '0x0000000000000000000000000000000000000000',
-            isCollateral: false,
-        },
-        KITE: {
-            address: '0xcDBAc5A71370047d5bA191cC6569f2914a4c2cd2',
-            decimals: 18,
-            symbol: 'KITE',
-            bytes32String: '',
-            collateralJoin: '',
-            collateralAuctionHouse: '',
-            isCollateral: false,
-        },
```

Comment:
Rename as below

KITE > ODG
HAI > OD
(not sure what TTM is)

Update the addresses once we have from web3 channel